### DISCUSSION TITLE
Please enable HDR to SDR conversion

### DISCUSSION THREAD

**Discussion by bubbleofmoon** [rustdesk/rustdesk#7652] [Feature Request]
I know many other remote control app also doesn't support it, But I want to know if it's possible.
I always has picture washed out problem on remote hdr windows machine from my mac.
I didn't test it with other system clients and I wonder if its mac problem or just app don't have a conversion feature.

**Comment by euphorah** [comment]
it should be possible for a setting that system wide disables HDR on connection then reenables upon disconnection

**Reply by pp-X** [reply]
how to set

**Comment by xlionjuan** [comment]
Just Win+Alt+B

**Comment by jnpatrick99** [comment]
Please fix it! Thank you. All other protocols including VNC work perfectly except RustDesk. I tried using HDR monitor on client, it helps a little but colors are still wrong.

**Comment by kndxhz** [comment]
i need too.

**Comment by shinkamui** [comment]
HDR is still a problem

**Comment by andmpel** [comment]
Please help

**Comment by dixtdf** [comment]
While using HDMI brightness adjustment software recently, I accidentally discovered that HDR brightness can be adjusted. After adjusting the brightness, the RustDesk view appears much more normal and no longer suffers from overexposure. I haven’t tried the DP interface yet, but it might also work.
https://github.com/xanderfrangos/twinkle-tray

<img width="516" height="270" alt="image" src="https://github.com/user-attachments/assets/69fe6117-eb60-4b5e-9b72-e749e1af0537" />

**Comment by ddel33t** [comment]
+++
